src/components/ImportJobs/components/ReturnToAssignJobs/ReturnToAssignJobs.test.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import { noop } from 'lodash';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+
+import '../../../../../test/jest/__mock__';
+
+import { FILE_STATUSES } from '../../../../utils';
+
+import {
+  renderWithFinalForm,
+  translationsProperties,
+} from '../../../../../test/jest/helpers';
+
+import { UploadingJobsContext } from '../../../UploadingJobsContextProvider';
+
+import { ReturnToAssignJobs } from './ReturnToAssignJobs';
+
+jest.mock('@folio/stripes-data-transfer-components', () => ({ Preloader: () => <span>Preloader</span> }));
+
+const mockOnResumeProp = jest.fn();
+
+const mockDeleteUploadDefinition = jest.fn(async () => Promise.reject(new Error('Something went wrong')));
+
+jest.spyOn(console, 'error').mockImplementation(() => {});
+
+const defaultContext = {
+  uploadDefinition: {
+    fileDefinitions: [
+      { status: FILE_STATUSES.UPLOADED },
+      { status: FILE_STATUSES.UPLOADING },
+    ],
+  },
+  updateUploadDefinition: noop,
+  deleteUploadDefinition: mockDeleteUploadDefinition,
+};
+
+const renderReturnToAssignJobs = () => {
+  const component = () => (
+    <UploadingJobsContext.Provider value={defaultContext}>
+      <ReturnToAssignJobs
+        onResume={mockOnResumeProp}
+        prohibitFilesUploading={false}
+      />
+    </UploadingJobsContext.Provider>
+  );
+
+  return renderWithIntl(renderWithFinalForm(component), translationsProperties);
```

Comment:
Do we need to render this with a form? It looks like no form used to render this component